### PR TITLE
Fixed empty page after deleting last portfolio item.

### PR DESCRIPTION
**Pull request by Hyperkid123**
### Changes
- if you delete all items from a page user is redirected to first page

empty-portfolio

### CODE REVIEW

File: src/smart-components/portfolio/portfolio.js [hunk 1]
```diff
@@ -68,9 +68,18 @@ const Portfolio = props => {
     .catch(() => dispatch({ type: 'setCopyInProgress', payload: false }));
   };
 
+  const handleEmptyPage = () => {
+    dispatch({ type: 'setIsFetching', payload: true });
+    return props.fetchPortfolioItemsWithPortfolio(props.match.params.id, defaultSettings)
```
Comment (lgalis) [inline src/smart-components/portfolio/portfolio.js]: We should keep the existing page limit here, and not switch to the default.

Comment (Hyperkid123) [reply src/smart-components/portfolio/portfolio.js]: Right switch to first page but keep the limit :+1:

**Comment by Hyperkid123** [comment]
@lgalis hm I was sure it was working.

But I've also discovered another issue with trying to keep current page. When deleting from multiple pages for instance. I think the correct way would be always fetch first page after deleting portfolio items. I think trying to keep current limit/offset is unfortunately not going to work when you start thinking about the edge cases.

**Comment by lgalis** [comment]
@Hyperkid123 - agree, going to the first page and keeping the page size would be right.

**Pull request merged**